transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -643,20 +643,19 @@ private void setUnwritable(boolean invokeLater) {
     }
 
     private void fireChannelWritabilityChanged(boolean invokeLater) {
-        final ChannelPipeline pipeline = channel.pipeline();
         if (invokeLater) {
             Runnable task = fireChannelWritabilityChangedTask;
             if (task == null) {
                 fireChannelWritabilityChangedTask = task = new Runnable() {
                     @Override
                     public void run() {
-                        pipeline.fireChannelWritabilityChanged();
+                        channel.channelWritabilityChanged();
```

Comment:
Nit, but this runnable can be a lambda or method reference.

Reply:
Actually I think we don't need this change at all for now